bancor_research/__init__.py

```diff
@@ -65,7 +67,7 @@ class DEFAULT:
         "decimals": DEFAULT.DECIMALS,
         "trading_fee": DEFAULT.TRADING_FEE,
         "bnt_funding_limit": DEFAULT.BNT_FUNDING_LIMIT,
-        "ep_vault_balance": DEFAULT.EP_VAULT_BALANCE,
+        "ep_vault_balance": Token(balance=DEFAULT.EP_VAULT_BALANCE),
```

Comment:
I think that we should apply this internally in function `init_protocol`, i.e., something like:
```
            # initialize tokens
            state.tokens[tkn_name] = Tokens(
                tkn_name=tkn_name,
                decimals=decimals,
                trading_fee=trading_fee,
                bnt_min_liquidity=bnt_min_liquidity,
                network_fee=network_fee,
                bnt_funding_limit=bnt_funding_limit,
                withdrawal_fee=withdrawal_fee,
                cooldown_time=cooldown_time,
                external_protection_vault=Token(balance=ep_vault_balance),
            )
```